tests/coordinator/test_assignors.py

```diff
@@ -1,27 +1,36 @@
 from collections import defaultdict
 from random import randint, sample
+from typing import Callable, Dict, Generator, Optional, Sequence, Set
+from unittest.mock import MagicMock
 
 import pytest
+from pytest_mock import MockerFixture
 
 from aiokafka.coordinator.assignors.range import RangePartitionAssignor
 from aiokafka.coordinator.assignors.roundrobin import RoundRobinPartitionAssignor
 from aiokafka.coordinator.assignors.sticky.sticky_assignor import (
     StickyPartitionAssignor,
 )
-from aiokafka.coordinator.protocol import ConsumerProtocolMemberAssignment
+from aiokafka.coordinator.protocol import (
+    ConsumerProtocolMemberAssignment,
+    ConsumerProtocolMemberMetadata,
+)
 from aiokafka.structs import TopicPartition
 
 
 @pytest.fixture(autouse=True)
-def reset_sticky_assignor():
+def reset_sticky_assignor() -> Generator[None, None, None]:
     yield
     StickyPartitionAssignor.member_assignment = None
     StickyPartitionAssignor.generation = -1
 
 
 def create_cluster(
-    mocker, topics, topics_partitions=None, topic_partitions_lambda=None
-):
+    mocker: MockerFixture,
+    topics: Set[str],
+    topics_partitions: Optional[Set[int]] = None,
+    topic_partitions_lambda: Optional[Callable[[str], Optional[Set[int]]]] = None,
+) -> MagicMock:
```

Comment:
Shouldn't we declare mocked type instead? Not sure whether this is correct though.

Reply:
I think, because we return Mock, we should use mock as an annotation.

Reply:
We may want to introspect the mock to check calls, so your annotation is correct